prow/config/jobs/istio.yaml

```diff
@@ -302,6 +302,21 @@ jobs:
       - name: INTEGRATION_TEST_FLAGS
         value: " --istio.test.retries=1 "
 
+  - name: integ-k8s-127
+    types: [postsubmit]
+    modifiers: [hidden]
+    command:
+      - entrypoint
+      - prow/integ-suite-kind.sh
+      - --node-image
+      - jacodelg/node:v1.27.0-rc.0
```

Comment:
@howardjohn can you either push your own or copy this to gcr.io?

Reply:
I already did, I had this same PR open but forgot to submit it :-) Should be in GCR